common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -76,6 +76,7 @@ eActionOp_CONTROL:
   ACTION_CONTROL_CLIENT_RX_RSSI_MEASUREMENT_START_NOTIFICATION: 104
   ACTION_CONTROL_CLIENT_RX_RSSI_MEASUREMENT_NOTIFICATION: 105
   ACTION_CONTROL_CLIENT_NO_RESPONSE_NOTIFICATION: 106
+  ACTION_CONTROL_CLIENT_NEW_IP_ADDRESS_NOTIFICATION: 107
```

Comment:
```suggestion
  ACTION_CONTROL_CLIENT_DHCP_NOTIFICATION: 107
```

Reply:
I think that "NEW_IP_ADDRESS" is good enough.
Today the only supported IP source is indeed a DHCP server, but maybe in the future we may allow setting the IP address via BML.